### PR TITLE
OCM-6514 | fix: Added more info to get config error message when config is not found

### PR DESCRIPTION
**Pull request by tirthct**
**Changed**
- Added more info to get config error message when config is not found

https://issues.redhat.com/browse/OCM-6514

### CODE REVIEW

File: cmd/config/get/get.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func PrintConfig(arg string) error {
 		if err != nil {
 			return fmt.Errorf("Failed to find config file location: %v", err)
 		}
-		return fmt.Errorf("Config file '%s' does not exist", loc)
+		return fmt.Errorf("Config file '%s' does not exist. Please run 'rosa login' command and try again", loc)
```
Comment (robpblake) [inline cmd/config/get/get.go:70]: @tirthct Nit: "Please run the `rosa login` command and try again."

Comment (tirthct) [reply cmd/config/get/get.go]: Fixed

**Review by robpblake — APPROVED** [review]
/lgtm

**Pull request merged**